### PR TITLE
Disable Differential Summaries

### PR DESCRIPTION
**Pull request by anthony-murphy**
related #4150 
related #4248

### CODE REVIEW

File: packages/runtime/runtime-utils/src/summarizerNode.ts [hunk 1]
```diff
@@ -599,7 +599,10 @@ export class SummarizerNode implements ISummarizerNode {
         private readonly initialSummary?: IInitialSummary,
     ) {
         this.canReuseHandle = config.canReuseHandle ?? true;
-        this.throwOnError = config.throwOnFailure ?? false;
+        // BUGBUG: Seeing issues with differential summaries.
+        // this will disable them, and throw instead
+        // why we continue to investigate
```
Comment (arinwt) [inline packages/runtime/runtime-utils/src/summarizerNode.ts:604]:
```suggestion
        // while we continue to investigate
```

File: packages/runtime/runtime-utils/src/summarizerNode.ts [hunk 2]
```diff
@@ -599,7 +599,10 @@ export class SummarizerNode implements ISummarizerNode {
...
+        this.throwOnError = config.throwOnFailure ?? true;
```
Comment (arinwt) [inline packages/runtime/runtime-utils/src/summarizerNode.ts:605]: I think we should either force to `true` here or change at all constructor call stacks.

Comment (vladsud) [reply packages/runtime/runtime-utils/src/summarizerNode.ts:605]: +1 to just set it to true unconditionally.

**Review by vladsud — APPROVED** [review]

**Review by arinwt — APPROVED** [review]

File: packages/runtime/runtime-definitions/src/summary.ts [hunk 3]
```diff
@@ -44,8 +44,11 @@ export interface ISummarizerNodeConfig {
      * attempt creating a summary that is a pointer ot the last acked summary
      * plus outstanding ops in case of internal summarize failure.
      * Defaults to false.
+     *
+     * BUG BUG: Disable config and default to true while we investigate problem
+     * with differential summaries
      */
-    readonly throwOnFailure?: boolean,
+    // readonly throwOnFailure?: boolean,
```
Comment (arinwt) [inline packages/runtime/runtime-definitions/src/summary.ts]: I don't think I would remove this, since it is part of the contract

**Pull request merged**